cedar-lean/Cedar/Thm/Authorization/Authorizer.lean

```diff
@@ -310,80 +300,56 @@ theorem mapM_asEntityUID_of_uid {uids : List EntityUID} :
   case nil =>
     rw [List.map_nil, List.mapM_nil]
     simp [pure, Except.pure]
-  case cons x xs h_ind =>
+  case cons x xs ih =>
     rw [List.map_cons, List.mapM_cons]
-    simp [pure, Except.pure, asEntityUID_of_uid, h_ind]
+    simp [pure, Except.pure, asEntityUID_of_uid, ih]
 
 /--
-  A generic lemma about the behavior of List.mapM' in the Except monad
+  Std has `Option.isSome_iff_exists`, but not this analogue for `Except`
 -/
-theorem mapM'_ok_iff_f_ok_on_all_elements {f : α → Except ε β} {list : List α} :
-  Except.isOk (list.mapM' f) ↔ ∀ x ∈ list, Except.isOk (f x)
```

Comment:
No longer needed, as #297's `List.mapM'_ok_implies_all_ok` is stronger

Reply:
Actually I think this is the stronger version since it's a bijection rather than an implication. Still good to remove if not needed though.

Reply:
I guess they're incomparable.  This version is stronger in that it's a bijection, but weaker in that it only promises `Except.isOk` while the other lemma promises things about the actual values.